### ISSUE TITLE
Bug: False values in repeated list cause spurious repeats

### ISSUE THREAD

**Issue by d-led** [ianka/mustache.tcl#3]
This seems to not work as expected:

```tcl
source mustache.tcl

set template {{{#items}}{{item}}{{#comma}},{{/comma}}{{/items}}{{#items}}{{item}}{{/items}}{{#items}}{{item}}{{#comma}},{{/comma}}{{/items}}
}

set data {
	items {{item a comma true} {item b}}
}

set result [::mustache::mustache $template $data]

# expected a,baba,b
# result: a,aa,baba,b
puts $result
```

test case: [bugs.yml](https://github.com/d-led/mustache.tcl/blob/bug1/tests/bugs.yml)

Added a test case, which fails: https://travis-ci.org/d-led/mustache.tcl#L157-L166

**Comment by d-led** [comment]
would you like to have the travis config/readme as a pull request?

**Comment by ianka** [comment]
I don't know what you mean by that.

**Comment by d-led** [comment]
in my fork I've added readme formatting: [README.md](https://github.com/d-led/mustache.tcl/blob/master/README.md) and a configuration for the travis-ci CI service to build and test the project on every commit/pull request: [.travis-yml](https://github.com/d-led/mustache.tcl/blob/master/.travis.yml). The build output [link](https://travis-ci.org/d-led/mustache.tcl#L157-L166) is the result of the build config. 

Pull request: I can propose my changes into your repository from my fork

**Comment by ianka** [comment]
Let's keep this separated from this bug report.

**Comment by d-led** [comment]
sure. I'd do it separately

**Comment by ianka** [comment]
I have found a simpler testcase which also fails and again, the whole problem arises from a much too naïve approach how repeated sections should be parsed. And this time it will really take me a while to fix this.

Thanks for reporting.

**Comment by d-led** [comment]
You're welcome! Thanks for the cool library

**Comment by ianka** [comment]
fixed in v1.1.3.2
